content/blog/esp32-bootstrapping/index.md

```diff
@@ -347,11 +423,88 @@ I (205) heap_runtime: ESP heap runtime init at 0x3fca4f70 size 273 kB.
 uart:~$
 ```
 
+
 ### MCUboot Espressif port (EP)
 
 To learn how to build the MCUboot Espressif Port, check out this [article](https://docs.mcuboot.com/readme-espressif.html)
 
-## Feature table
+
+### AMP enabled sample code
+
+AMP builds require several images to be built and flashed onto a target SoC. Let's use ESP32-S3 as our test platform and demonstrate the sysbuild capabilities on the sample code provided by the Zephyr RTOS sources.
+The Zephyr sample code (`samples/drivers/ipm/ipm_esp32`) uses IPM (Inter-Processor Mailbox) to demonstrate a simple two-way communication between the `PRO_CPU` core and `APP_CPU` core. Images for both CPU cores must be loaded using the MCUboot.
+Note that there is no support for running AMP if the Simple Boot mechanism is used.
+
+Building and flashing a complete set of images running the IPM sample:
```

Comment:
How about this:

```suggestion
Now we can build and flash a complete set of images running the IPM sample:
```

Reply:
thx! fixed

Reply:
Thanks @f-hollow, I've fixed all remaining findings.